proknow-sdk/Patient/Entities/EntityScorecards.cs

```diff
@@ -73,7 +73,7 @@ public EntityScorecards(ProKnowApi proKnow, string workspaceId, string entityId)
             var customMetricIdsAndObjectives = resolvedCustomMetrics.Select(c => c.ConvertToScorecardSchema()).ToList();
 
             // Request the creation
-            var route = $"/workspaces/{_workspaceId}/patient/{_entityId}/metrics/sets";
+            var route = $"/workspaces/{_workspaceId}/patients/{_entityId}/metrics/sets";
```

Comment:
In the constructor of this class, 
* Change the 3rd argument from `entityId` to `patientId`
* Update the XML comments to match
* Change the private var name from `_entityId` to `_patientId`